packages/shared/src/utilities/color/isRgbaLongFormat.ts

```diff
@@ -0,0 +1,6 @@
+import { Color } from '@frontify/fondue';
+
+export const isRgbaLongFormat = (value: Color) => {
```

Comment:
Nice 👍🏻  could you also use this in the `toShortRgba` utility? And maybe add some tests?

Reply:
removed the file again, as I actually don't need the isRgbaLongFormat.ts in this case